classes/Utils.php

```diff
@@ -4063,7 +4063,7 @@ public function get_tutor_user( $user_id ) {
 	 *
 	 * @since 1.0.0
```

Comment:
@since 3.6.0 param $is_course_object is added and update param $course_id to $object_id